component-runtime-impl/src/main/java/org/talend/sdk/component/runtime/record/SchemaImpl.java

```diff
@@ -152,10 +159,19 @@ private static String sanitizeConnectionName(final String name) {
 
             @Override
             public Builder withName(final String name) {
+                // can't do that now as AvroSchemas.sanitizeConnectionName is called before current method
+                // and not so easy to remove AvroSchemas.sanitizeConnectionName
+                /// withLabel(name);
```

Comment:
some client use this code like this : withName(AvroSchemas.sanitizeConnectionName
(name)), i need to consider more if refactor, more risk, so do like this

Reply:
I think it's a mistake in those connectors and should not take into account. It will change nothing if withLabel is called when name != sanitizedName in this method, for those connectors.

They wil call :
- withName(AvroSchemas.sanitizeConnectionName("1field"))
- withName("_1field")
- in withName : givenName == sanitized(givenName) since already done so no withLabele

But for all other connectors that call withName("1field");
in withName : givenName/1field != sanitiedName/_1field so we should call withLabel(givenName)

You think it is too early ?